stable/gcpdisk-csi-driver/templates/csi-gcpdisk-controller.yaml

```diff
@@ -88,6 +88,10 @@ spec:
           args:
             - --v=5
             - --csi-address=$(ADDRESS)
+            - --metrics-address=:22013
```

Comment:
if these ports are the default ones, I don't see much interest on adding them

Reply:
see https://github.com/mesosphere/charts/pull/584/files#r430020207